PumaRhino/RawInitialShape.h

```diff
@@ -19,13 +19,11 @@
 
 #pragma once
 
-#include "MaterialAttribute.h"
-#include "ReportAttribute.h"
-#include "RhinoCallbacks.h"
-#include "utils.h"
-
+#include <string>
 #include <vector>
 
+class ON_Mesh;
```

Comment:
all sites which include RawInitialShape.h also include stdafx.h before which includes ON_Mesh, so the type is known. I still consider it good practice to declare (or include) all types used to avoid surprises when the header is included somewhere else.